### PR TITLE
Add venv, IDE, and Cythonized files to gitignore

### PR DESCRIPTION
**Pull request by chinandrew**
Some gitignore additions so build and environment files don't get committed.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,11 @@
 **/.pytest_cache
 **/.ipynb_checkpoints
 bayesbridge/reg_coef_sampler/hamiltonian_monte_carlo/tests
+.eggs/
+.idea/
+env/
+bayesbridge.egg-info/
+bayesbridge/random/polya_gamma/polya_gamma.c
+bayesbridge/random/polya_gamma/polya_gamma.cpython-39-x86_64-linux-gnu.so
```
Comment (aki-nishimura) [inline .gitignore:12]: Maybe 
`bayesbridge/random/polya_gamma/polya_gamma.*.so` 
`bayesbridge/random/tilted_stable/tilted_stable.*.so`
is better?

**Review by aki-nishimura — COMMENTED** [review]
Perhaps better to use globbing? (With apology for being nitpicky.)

**Comment by chinandrew** [comment]
> Perhaps better to use globbing? (With apology for being nitpicky.)

Not a nitpick at all, globbing makes much more sense here. Thanks!

File: .gitignore (hunk 1 above)
Comment (chinandrew) [reply .gitignore]: Updated

**Pull request merged**